src/unity/lib/annotation/utils.cpp

```diff
@@ -1,10 +1,11 @@
 #include "utils.hpp"
 
 bool turi::annotate::isInteger(std::string s) {
-	if(s.empty() || ((!isdigit(s[0])) && (s[0] != '-') && (s[0] != '+'))) return false;
```

Comment:
Our these and the other annotator changes intensionally part of this PR?

Reply:
Yes (though only the non-whitespace changes were intended -- that was my editor trying to be "helpful"). There was a compiler (warning-as-)error on unused return value; I modified this code to suppress the error.